dev/README_RELEASE_AIRFLOW.md

```diff
@@ -664,7 +662,9 @@ Building the image is triggered by running the
 When you trigger it you need to pass:
 
 * Airflow Version
-* Optional "true" in skip latest field if you do not want to retag the latest image
+* Optional "true" in skip latest field if you do not want to re-tag the latest image
```

Comment:
Maybe we can add something like "`rc` suffix and other pre-release version will automatically set the `skip latest` to true".

(Or a different wording, but I remembered wondering if I should skip or not for rc, and had to check the code)

Reply:
Good idea. Added.